.codegen/resolvers.go.tmpl

```diff
@@ -0,0 +1,54 @@
+// Code generated from OpenAPI specs by Databricks SDK Generator. DO NOT EDIT.
+
+package resolvers
+
+{{ $allowlist :=
+	list
+		"alerts"
+        "clusters"
+        "cluster-policies"
+        "clusters"
+        "dashboards"
+        "instance-pools"
+        "jobs"
+        "metastores"
+        "pipelines"
+        "queries"
+        "warehouses"
```

Comment:
These are great! But could you also support service principals and webhooks as seen in https://github.com/databricks/universe/blob/master/serverless-smoke-tests/databricks.yml?

Reply:
@andrewnester could you still take a look at this? Are those resource types supported or not?

Reply:
@lennartkats-db added service principals but it seems like webhook notifications does not have an API (at least the one defined in OpenAPI) hence no support yet.

Reply:
I see. It's disappointing we can't support webhooks yet, but we should indeed do that with a proper public API. Let's push on getting that API support. I tagged you on a related thread.